tests/accuracy/test_global_correction.py

```diff
@@ -157,6 +160,54 @@ def test_lhc_global_correct(tmp_path, model_inj_beams, orientation):
         diff_rms_prev = diff_rms
 
 
+@pytest.mark.basic
+@pytest.mark.parametrize('dpp', (2.5e-4, -1e-4))
+def test_lhc_global_correct_dpp(tmp_path: Path, model_inj_beams: DotDict, dpp: float):
+    response_path = tmp_path / "full_response_dpp.h5"
+    response_dict = create_response(
+        outfile_path = response_path,
+        variable_categories=[ORBIT_DPP],
+        delta_k=2e-5,
+        **model_inj_beams,
+    )
+
+    # Basic check if response was created correctly
+    for key in response_dict.keys():
+        assert ORBIT_DPP in response_dict[key].columns
+
+    # create fake measurement from previously created model
+    dpp_path = CORRECTION_INPUTS / "deltap" / f"twiss_dpp_{dpp:.1e}_B{model_inj_beams.beam}.dat"
+    model_df = tfs.read(dpp_path, index=NAME)
+    fake_measurement(
+        twiss = model_df,
+        parameters = [f"{PHASE}X", f"{PHASE}Y"],
+        outputdir = tmp_path,
+    )
+
+    # See if the simulated dpp can be recreated
+    for update_response in [True, False]:
+        diff = np.inf
+        for iteration in range(1, 4): # Must be at least 3 to test convergence
+            global_correction(
+                meas_dir = tmp_path,
+                output_dir = tmp_path,
+                fullresponse_path = response_path,
+                variable_categories=[ORBIT_DPP],
+                optics_params = [f"{PHASE}X", f"{PHASE}Y"],
+                iterations=iteration,
+                update_response=update_response,
+                **model_inj_beams,
+            )
+            result = tfs.read(tmp_path / "changeparameters_iter.tfs", index=NAME)
+            
+            # Check if the output is correct within 5% (Beam 2 is not as accurate)
+            rtol = 5e-2 if iteration == 1 else 2e-2
+            assert np.isclose(dpp, -result[DELTA][ORBIT_DPP], rtol=rtol), f"Expected {dpp}, got {result[DELTA][ORBIT_DPP]}, diff: {dpp + result[DELTA][ORBIT_DPP]}, iteration: {iteration}"
```

Comment:
Which order of magnitude do we expect for the values here? The default of `atol` could be an issue.

Reply:
Ah nevermind the values are given in the fixture and much higher than the `1e-08` of `atol`, my bad.